contracts/MinGov.sol

```diff
@@ -69,12 +69,12 @@ contract MinGov is Ownable {
             getSig(prop.msgData) == 0x3659cfe6 // upgradeTo(address)
           ) {
             // this changes proxy parameters 
-            rv = prop.subject.call(prop.msgData);
+            (rv, ) = prop.subject.call(prop.msgData);
             // use this for 0.5
             // (rv, ) = prop.subject.call(prop.msgData);
           } else {
             // this changes governance parameters to the implementation
-            rv = AdminableProxy(prop.subject).applyProposal(prop.msgData);
+            rv = AdminableProxy(address(uint160(prop.subject))).applyProposal(prop.msgData);
```

Comment:
same here.. `prop.subject` is an address already, no?

Reply:
there is a difference between address and address payable now, only second one can be transfered to. in can only be cast from uint160. 🤷‍♂️